package.sh

```diff
@@ -170,18 +181,34 @@ make_dir_tree() {
     fi
 }
 
-
 # do_build builds the code. The version and commit must be passed in.
 do_build() {
     for b in ${BINS[*]}; do
         rm -f $GOPATH_INSTALL/bin/$b
     done
+    
+    if [ ! -z "$WORKING_DIR" ]; then
```

Comment:
What is the idea of this? I wanted the packaging script to be super careful, so it explicitly exits if the local tree is dirty. We don't want any screw-ups during packaging, so one must have a clean tree before packaging.

Thoughts?